modules/osd-release-notes-Q4-2025.adoc

```diff
@@ -10,7 +10,14 @@
 For more information, see link:http://docs.redhat.com/en/documentation/openshift_dedicated/4/html-single/openshift_dedicated_clusters_on_google_cloud/index#wif-configuration-update_osd-creating-a-cluster-on-gcp-with-workload-identity-federation[Updating a Workload Identity Federation configuration].
 
 * **Required API services table updated.**
-The _Required API services_ table within the _Required customer procedure_ guide has been updated to restore APIs that were previously removed due to a bug. These APIs are required for new {product-title} on {GCP} cluster creation. For more information, see link:https://docs.redhat.com/en/documentation/openshift_dedicated/4/html/planning_your_environment/gcp-ccs#ccs-gcp-customer-procedure_gcp-ccs[Required customer procedure].
+The _Required API services_ table within the _Required customer procedure_ guide has been updated to restore APIs that were previously removed due to a bug. These APIs are required for new {product-title} on {GCP} cluster creation.
++
+The APIs that were restored are:
++
+** link:https://cloud.google.com/apis/docs/overview[Google Cloud APIs]
+** link:https://cloud.google.com/firewall/docs/reference/network-security/rest[Network Security API]
++
+For more information, see the _Required API services_ table in the link:https://docs.redhat.com/en/documentation/openshift_dedicated/4/html/planning_your_environment/gcp-ccs#ccs-gcp-customer-procedure_gcp-ccs[Required customer procedure].
```

Comment:
This is a really small nit, but this line is indented as if it were part of the bulleted list. That's fine if that's what you intended, but if not, you can delete the list continuation (`+`) and just leave a line break between the list and the link.